org.adempiere.base/src/org/compiere/util/DBReadReplica.java

```diff
@@ -168,6 +168,8 @@ private static void setUserPass() {
 					m_pass = value;
 				}
 			}
+			if (m_pass == null && MSystem.isSecureProps())
+				m_pass = Ini.getVar("ADEMPIERE_DB_PASSWORD");
```

Comment:
this is a questionable one - now all read replica must use the same password as the master db ?